components/Button/Button.tsx

```diff
@@ -0,0 +1,44 @@
+import {ViewStyle, Pressable, TextStyle, StyleSheet, Text} from 'react-native';
+import React from 'react';
+
+const Button = ({
+  text,
+  buttonStyle,
+  textStyle,
+  disabled = false,
+  onPress,
+}: {
+  text?: string;
```

Comment:
text and onPress shouldn't be optional since button always has title and onPress